app/controllers/idv/sessions_controller.rb

```diff
@@ -15,32 +15,36 @@ class SessionsController < ApplicationController
 
     def new
       user_session[:context] = 'idv'
-      @view_model = view_model
-      @view_model.selected_state = user_session[:idv_jurisdiction]
+      @idv_form = idv_form
+      @selected_state = user_session[:idv_jurisdiction]
       analytics.track_event(Analytics::IDV_BASIC_INFO_VISIT)
     end
 
     def create
-      result = idv_form.submit(profile_params)
+      @idv_form = idv_form
```

Comment:
Similar comment as above. `idv_form` (line 99) uses an instance variable `@_idv_form` to store this. If you do need `@idv_form`, then perhaps change `@_idv_form` to `@idv_form` and then use `idv_form` here to make sure it's defined (in case you want it in a view).